src/app/core/facades/app.facade.ts

```diff
@@ -2,24 +2,30 @@ import { Injectable } from '@angular/core';
 import { NavigationCancel, NavigationEnd, NavigationStart, Router } from '@angular/router';
 import { Store, select } from '@ngrx/store';
 import { combineLatest, merge } from 'rxjs';
-import { filter, map, mapTo, shareReplay, startWith } from 'rxjs/operators';
+import { filter, map, mapTo, shareReplay, startWith, withLatestFrom } from 'rxjs/operators';
 
-import { getAvailableLocales, getCurrentLocale, getDeviceType, getICMBaseURL } from 'ish-core/store/configuration';
+import {
+  getAvailableLocales,
+  getCurrentLocale,
+  getDeviceType,
+  getICMBaseURL,
+  getServerConfigParameter,
+} from 'ish-core/store/configuration';
 import { LoadCountries, getAllCountries, getCountriesLoading } from 'ish-core/store/countries';
 import { getGeneralError, getGeneralErrorType } from 'ish-core/store/error';
 import { LoadRegions, getRegionsByCountryCode } from 'ish-core/store/regions';
+import { getLoggedInCustomer } from 'ish-core/store/user';
 import { getBreadcrumbData, getHeaderType, getWrapperClass, isStickyHeader } from 'ish-core/store/viewconf';
 
 @Injectable({ providedIn: 'root' })
 export class AppFacade {
-  icmBaseUrl: string;
-
   constructor(private store: Store, private router: Router) {
     // tslint:disable-next-line: rxjs-no-ignored-subscribe
     this.routingInProgress$.subscribe();
 
     store.pipe(select(getICMBaseURL)).subscribe(icmBaseUrl => (this.icmBaseUrl = icmBaseUrl));
   }
+  icmBaseUrl: string;
```

Comment:
I think some auto-formatting positioned this here, I'll investigate.. 🤔

Reply:
I leave this as it is